### PR TITLE
snake_case & camelCase variable

### PR DESCRIPTION
**Pull request by Roh-Patil**

### CODE REVIEW

File: packages/eslint/variable_name_check.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		schema: [],
+	},
+	create(context) {
+		return {
+			VariableDeclarator(node) {
+				if (
+					node.parent.kind === 'let'
+                    && node.type === 'VariableDeclarator'
+				) {
+					const variableName = node.id.name;
```
Comment (Rajatuba-Cogoport) [inline packages/eslint/variable_name_check.js]: Id?. Name

Comment (Rajatuba-Cogoport) [reply packages/eslint/variable_name_check.js]: || undefined

File: packages/eslint/variable_name_check.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+					if (
```
Comment (Rajatuba-Cogoport) [inline packages/eslint/variable_name_check.js]: Add one more conditon, check variable name is present or not

**Review by Rajatuba-Cogoport — CHANGES_REQUESTED** [review]

File: packages/eslint/variable_name_check.js (hunk 2 above)
Comment (Roh-Patil) [reply packages/eslint/variable_name_check.js]: done

File: packages/eslint/variable_name_check.js (hunk 1 above)
Comment (Roh-Patil) [reply packages/eslint/variable_name_check.js]: done

**Review by Rajatuba-Cogoport — APPROVED** [review]

**Review by Rajatuba-Cogoport — APPROVED** [review]

**Pull request merged**